extractor/src/test/java/org/schabi/newpipe/extractor/services/DefaultStreamExtractorTest.java

```diff
@@ -45,6 +45,7 @@
     public abstract String expectedUploaderName();
     public abstract String expectedUploaderUrl();
     public boolean expectedUploaderVerified() { return false; }
+    public long expectedUploaderSubscriberCountAtLeast() { return -1; }
```

Comment:
This test isn't the best: if somebody forgets to override it, the related test will always pass, so it wouldn't be testing anything. The other methods with default values provide only one possible value, so the related test will pass only if the value is the default one or otherwise fail (hence notifying the user that he should override the default function). In this case this doesn't happen. Here's my suggestion (1):
```suggestion
    public long expectedUploaderSubscriberCountAtLeast() { return UNKNOWN_SUBSCRIBER_COUNT; }
```

Reply:
> This test isn't the best: if somebody forgets to override it, the related test will always pass, so it wouldn't be testing anything.

That's intended, I didn't want to modify all existing tests by looking for values for each video, hence why.

Reply:
Oh, would it be a problem for you to solve the tests? The CI already tells you all of the expected values: https://github.com/TeamNewPipe/NewPipeExtractor/runs/5284419766

Reply:
Done!